front/src/common/RollingStockSelector/RollingStock2Img.jsx

```
@@ -18,9 +18,12 @@ function RollingStock2Img(props) {
     if (!defaultLivery?.id) return;
 
     try {
-      const image = await get(`/rolling_stock/${rollingStock.id}/livery/${defaultLivery.id}/`, {
-        responseType: 'blob',
-      });
+      const image = await get(
+        `/editoast/rolling_stock/${rollingStock.id}/livery/${defaultLivery.id}/`,
+        {
+          responseType: 'blob',
+        }
+      );
       if (image) setImageUrl(URL.createObjectURL(image));
     } catch (e) {
       /* empty */
```

No empty catch error (at least put a `console.error`, thx !)